bettercli/option.py

```
@@ -1,88 +1,110 @@
+    @property
+    def has_default(self) -> 'bool':
+        logger.debug("Positional_option.has_default")
+        return self.default is not None
 
-class Positional_option(Option[T]):
-    def __init__(self, name:'str', type=None, *, default=None, length=None, min_length=None, max_length=None):
-        logger.debug(f"Positional_option.__init__: {name=} {type=} {default=} {length=} {min_length=} {max_length=}")
-        super().__init__(name, type, default=default, length=length, min_length=min_length, max_length=max_length)
\ No newline at end of file
+
+    def validate(self, option: 'str', cur_option: 'list[str]'=[]) -> 'tuple[t.Union[InvalidType, InvalidLength, t.Literal[True]], bool]':
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)